lib/cli/kit/base_command.rb

```diff
@@ -0,0 +1,43 @@
+require 'cli/kit'
+
+module CLI
+  module Kit
+    class BaseCommand
```

Comment:
This is `Dev::Command`, more or less. I was able to extract a reasonable amount of it to here and subclass this over there.